sys-libs/libnbd/libnbd-1.14.2.ebuild

```diff
@@ -50,7 +50,6 @@ src_configure() {
 		$(use_enable ocaml)
 		$(use_enable python)
 		$(use_with gnutls)
-		$(use_with uri-support libxml2)
```

Comment:
You need to pass --with-uri-support unconditionally.

Reply:
fix